acsys/__init__.py

```diff
@@ -777,10 +778,9 @@ async def ping(self, node):
         try:
             await self.request_reply(f'ACNET@{node}', b'\x00\x00', timeout=250)
             return True
+        except status.AcnetRequestTimeOutQueuedAtDestination as exception:
```

Comment:
This should be

```python
        except status.AcnetRequestTimeOutQueuedAtDestination:
```

Reply:
Good point, I'm fixing them among several other redundant `except / raise`s